client/src/components/InputField.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+const InputField = (probs) => {
```

Comment:
You don't have probs, just props ;).
properties => props.

Happens multiple times.